src/content/docs/new-relic-solutions/increase-user-engagement.mdx

```diff
@@ -0,0 +1,196 @@
+
+## Objectives [#objectives]
+
+This doc helps you set up and understand your data, so you can empower your team to take control of your customer experience. In this tutorial, you will:
+
+* Set up monitoring for two pages with website performance monitoring 
+* Learn about the data collected from WPM
+
+## Set up WPM [#setup]
+<Steps>
+    <Step>
+    ### Get data without installation
+
+    To access WPM, go to **[one.newrelic.com > Website performance monitoring](https://one.newrelic.com/website-grader)**.
+
+    </Step>
+    <Step>
+        ### Choose your URL 
+
+    WPM requires no installation or set up. Once you've chosen pages to monitor, New Relic deploys a set of monitors from different servers around the globe that check each page for website availability, broken links, performance, and SSL certificate expiration. To get started, we recommend choosing these pages:
+
+    * Your homepage.
+    * Pages with critical assets, like important videos, images, or links.
+    * Pages critical to your customer funnel
+
+New Relic will take a few minutes to deploy the monitors, then you'll start receiving data about your website. Keep in mind that WPM doesn't check all pages across your site, only the ones you've manually added.
+
+    </Step>
+    <Step>
+        ### Add Google PageSpeed API key 
+
+When you set up WPM for the first time, New Relic gives you complementary core web vitals via Google PageSpeed. To continue receiving your core web vitals, you need to set up PageSpeed with your own account.
+
+<img
+    title=""
+    alt=""
+    src={pageSpeedAPI}
+/>
+
+    </Step>
+
+    <Step>
+        ### Put your data in context
+
+Once your monitors report data to New Relic, your WPM page displays metrics that can help you improve user experience. WPM gives you four kinds of data:
+
+<CollapserGroup>
+    <Collapser
+        id="ping"
+        title="Check page availability"
+    >
+<img
+    title="View your page availability"
+    alt="View page availability"
+    src={pingWPM}
+/>
+
+The first section of WPM shows you ping data. This kind of data is an availability check, confirming basic site functionalities:
+
+* **Site availability**. This reports whether your page has loaded at all. If it hasn't, there might be an issue with your servers or domain.
+* **Page load time**. This collects the median speed for how long it takes your page to load all of its assets. This data gets further broken down by content type and size.
+* **SSL Validity**. This tells you whether your SSL certificate is valid, expiring, or expired. It'll include the date you need to renew your SSL certificate as well.
+* **Broken links**. After testing all the links on your page, it'll report back which links are broken. 
+
+Because of how WPM collects data, these checks update as often as you've configured your deployment intervals. If something changes between the times you've manually checked your page, your monitors will report that change and alert you.
```

Comment:
"deployment intervals" almost sounds like a config variable on our end. Do we mean "the deployment intervals for your web application in your environment"?

Reply:
It's how often you've set your synthetics to deploy, which is limited by how much you want to pay per month for the experience. I'll rework this